### ISSUE TITLE
DKB-pushTAN

### ISSUE THREAD

**Issue by dnl911** [n00neimp0rtant/xCon-Issues#448]
App from a german bank to get codes for transactions.

Does not work with v40 Beta 1.

Sorry for german text on the screenshot. Like I said, the app is from a german bank.

**Comment by gomble** [comment]
Need Bypass Jailbreak for this too :/

**Comment by JayJack** [comment]
Same here :-(

**Comment by gomble** [comment]
The app pushtan from sparkasse is simmiular to this app.

**Comment by JayJack** [comment]
@gomble: True. Maybe both apps use the same technique for detecting jailbreak.

**Comment by gomble** [comment]
I hope the xcon devloper will include this apps to xcon :(

 

Von: JayJack [mailto:notifications@github.com] 
Gesendet: Montag, 9. Februar 2015 21:20
An: n00neimp0rtant/xCon-Issues
Cc: gomble
Betreff: Re: [xCon-Issues] DKB-pushTAN (#448)

 

@gomble <https://github.com/gomble> : True. Maybe both apps use the same technique for detecting jailbreak.

**Comment by Wepamat** [comment]
Same here, would be greatly appreciated to see support coming :+1:

**Comment by barkley82** [comment]
Not Working with xcon40 Beta 2 :( .. I hope so.

**Comment by gomble** [comment]
-.- there must be a solution

**Comment by kuao** [comment]
I hope this is getting some attention.

**Comment by Bensk1** [comment]
+1

**Comment by gomble** [comment]
?

**Comment by matthias-springer** [comment]
+1

**Comment by rhlff** [comment]
+1

**Comment by mathebox** [comment]
+1

**Comment by aaraT** [comment]
+1 a thousand times over!

**Comment by gomble** [comment]
+1

**Comment by jo2h2** [comment]
Would love to have this included as well - thanks !!

**Comment by ingotri** [comment]
me too!

**Comment by necrevistonnezr** [comment]
+ 1

**Comment by drreen** [comment]
+1

**Comment by dieterhatnkleinen** [comment]
still no solution?

**Comment by flocla** [comment]
id even pay for a workaround:D

**Comment by motokito** [comment]
I need the solution too. NEED HELP PLS !!!

**Comment by trueter** [comment]
+1

**Comment by legendaryApps** [comment]
definitely need a fix for this! There's so many people affected

**Comment by twdnhfr** [comment]
+1 👍

**Comment by motokito** [comment]
I have commented since 2 week, but no one of developer will support this.
Please support this issue. I will very thx about it, because i must need this tweak 😢

**Comment by SebastianRasch** [comment]
DKB's new 3D Secure system replaced "Verified by Visa" now so you have to use another DKB app with the same detection method, called "DKB-Card-Secure". You actually have to use this app now to use your credit card online, even when you're at your desktop computer (or you can choose the insecure SMS-Tan but you have to pay for every SMS you receive).
I really hope there will be a JB bypass some day, otherwise I can't use DKB anymore to pay online.
